CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to the Packer template for creating GraphDB AMIs will be documented in this file.
 
+## [1.1.0] - 2023-09-21
+
+- Added parallel building of `arm64` and `amd63` based AMIs
+- Added AWS cli to be installed based on the architecture
+- Added new variables for `arm64` instance type and source AMI  
+- Renamed `build_instance_type` and `build_instance_type` variables
```

Comment:
Update all references, check the readme

Reply:
Done

Reply:
There are still some references, see readme line 66